LeetCode/vAnil/Practice/PracticeQuestions/ReverseWordInString.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ReverseWordInString.h"
+using namespace std;
+
+PRACTICEQUESTIONS_API std::string ReverseWordsInString(std::string& str)
+{
+	// remove prefix spaces
+	size_t len = str.length();
+	size_t count = 0;
+	for (size_t i = 0; i < len; i++)
+	{
+		if (str[i] == ' ')
+		{
+			count++;
+		}
+		else
+			break;
+	}
+	str.erase(0, count);
```

Comment:
out of range subscript error if str is ""

Reply:
Fixed it.